src/legacy/ui/public/index_patterns/_field.ts

```diff
@@ -157,6 +157,7 @@ export class Field implements FieldType {
 
     // computed values
     obj.comp('indexPattern', indexPattern);
+    this.displayName = '';
     obj.comp('displayName', shortDotsEnable ? shortenDottedString(spec.name) : spec.name);
```

Comment:
@ppisljar, @lukeelmers, @timroes 
Since `displayName` property is set anyway (`name` is required, `displayName` is based on it), should it be required in that case? 

to be honest I need `displayName` to sort and display in Field combo box. I wonder if it's applicable just to make it required.